src/rt/cover.h

```diff
@@ -106,27 +109,37 @@ typedef enum {
 } cover_dump_t;
 
 typedef enum {
-   COVER_MASK_STMT   = (1 << 0),
-   COVER_MASK_BRANCH = (1 << 1),
-   COVER_MASK_TOGGLE = (1 << 2),
+   COVER_MASK_STMT                        = (1 << 0),
+   COVER_MASK_BRANCH                      = (1 << 1),
+   COVER_MASK_TOGGLE                      = (1 << 2),
+   COVER_MASK_TOGGLE_COUNT_FROM_UNDEFINED = (1 << 8),
+   COVER_MASK_TOGGLE_COUNT_FROM_TO_Z      = (1 << 9),
+   COVER_MASK_TOGGLE_IGNORE_MEMS          = (1 << 10),
+   COVER_MASK_TOGGLE_IGNORE_ARRAYS_FROM   = (1 << 11)
 } cover_mask_t;
 
-#define COVER_MASK_ALL (COVER_MASK_STMT | COVER_MASK_BRANCH | COVER_MASK_TOGGLE)
+#define COVER_MASK_ALL_TYPES (COVER_MASK_STMT | COVER_MASK_BRANCH | COVER_MASK_TOGGLE)
```

Comment:
Why change this? It seems to me that `COVER_MASK_ALL` is both shorter and clearer (the other masks don't have `TYPE` in them).

Reply:
I changed it back. The idea was to say by `TYPE` that this is mask with all coverage types, not necessarily
all coverage options.